include/swift/RemoteInspection/TypeRefBuilder.h

```diff
@@ -38,7 +39,8 @@ namespace reflection {
 
 using remote::RemoteRef;
 
-template <typename Runtime> class ReflectionContext;
+template <typename Runtime>
+class ReflectionContext;
```

Comment:
Can you commit a separate patch that just runs clang-format on this file? It would reduce the size of the diff considerably.

Reply:
There's already a commit that just clang formats this file

Reply:
There are three NFC commits and one commit with the actual changes

Reply:
You're right of course!